exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,22 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if(number<0){
```

Comment:
Excellent Cuneyt.  I just  want to  add one thing like the above.  To use  the if condition is not necessary for this exercise.
like this:
```
function isNegative(number) {
  return number < 0;
}
```